### ISSUE TITLE
[Feature Request] WatchList Tokens via config txt/ini

### ISSUE THREAD

**Issue by RhanCandia** [Goldenfreddy0703/Otaku#302]
Hello. I am on Linux and Kodi in Linux just don’t allow pasting. So getting the Anilist token from the browser to Kodi is a challenge I have’t solved yet. 

Would it be possible to have the tokens written on a txt or ini file instead? Can this be a feature? Or is this already possible?

Thanks!

**Comment by RhanCandia** [comment]
I went to the Kodi usedata folder and found the addon data folder for Otaku. I went ahead and manually inserted my username and token in the XML file and enabled Anilist. I am testing it now. I’ll close this ticket if it works.

**Comment by RhanCandia** [comment]
Manually setting the username and token in the settings xml didn’t work.

I just moved my tracking to Kitsu instead. And that worked easier. 

Closing this now.

**Comment by Goldenfreddy0703** [comment]
Hey so I thought about this too and thought it would be a good idea to export and import xml files from another device sense copy and pasting is hard for Linux and ios. This issue is technically a duplicate of #262 but we do plan to add an easier way in the future. Thanks for posting this.

**Comment by RhanCandia** [comment]
> Hey so I thought about this too and thought it would be a good idea to export and import xml files from another device sense copy and pasting is hard for Linux and ios. This issue is technically a duplicate of #262 but we do plan to add an easier way in the future. Thanks for posting this.

Thank you! I will follow that issue. I would really like it if I can use Anilist and authenticate in an easier way. I’ll try Kitsu for now.

**Comment by Goldenfreddy0703** [comment]
I think there is a workaround if you would like to use anilist right now, find your phone or any device and download the Kore app that kodi provides and you can have it as a remote and send text from your phone to kodi. 

https://play.google.com/store/apps/details?id=org.xbmc.kore

You can also try going into the kodi service settings and utilize the control category which will allow you to remote control via HTTP which will also allow to send text. Hope this helps

**Comment by RhanCandia** [comment]
Freddy, I tried AniList once again. I managed to make it work. 

So I went back to the settings.xml of the Otaku addon and put my AniList token and username there again. It works now. 

What I did wrong earlier is I put my email instead of the username.  It should be the username then the token. 

It works now. Manually adding the token to the Otaku settings.xml. 

That’s still not user friendly for most people. But that works for now. Thanks again!
